test/standalone/windows_spawn/main.zig

```diff
@@ -39,6 +40,9 @@ pub fn main() anyerror!void {
     // No PATH, so it should fail to find anything not in the cwd
     try testExecError(error.FileNotFound, allocator, "something_missing");
 
+    // Regression test - make sure we don't get error.BadPath traversing out of cwd with a relative path
```

Comment:
Minor nitpick: having `regression test` here doesn't provide much extra info
```suggestion
    // make sure we don't get error.BadPath traversing out of cwd with a relative path
```

(same goes for the comment in `testSpawnRelative`)